cli/develop.js

```diff
@@ -92,16 +95,16 @@ const run = (args) => {
     utils.log("---------------------------------------------------\n\n");
   }).on('error', (error) => {
     if (error.code === 'EADDRINUSE') {
-      utils.error(esapi.encoder.encodeForJavaScript(`Port ${app.get('port')} is currently in use by another program.
-      You must either close that program or specify a different port by setting the shell variable "$PORT". Note that on MacOS / Linux ${chalk.yellow(`lsof -n -i :${app.get('port')} | grep LISTEN`)} should identify the process currently using the port.`));
+      utils.error(`Port ${app.get('port')} is currently in use by another program.
```

Comment:
all the `esapi.encoder.encodeForJavaScript` statements were added by us to fix a security scan warning. It looks like you undid our security changes

Reply:
@atharvar28 removed this in different other places.. I just applied the same thing